.github/workflows/ena-submission-image.yaml

```diff
@@ -36,6 +36,13 @@ jobs:
       packages: write
       checks: read
     steps:
+      - name: Wait for submitToEnaProduction check
```

Comment:
I would say we just make the config check part of the image workflow - it's kind of directly related to it, and would save on the need to use this wait on check action in yet one more place.

Reply:
From my understanding the ena-submission-image workflow runs in the docker image, so it does not have access to the files in the kubernetes folder (I tried this before).

Reply:
The workflow is just a GitHub action that runs in a GitHub workflow runner. After the checkout step below it will have all the files in the repository.

Reply:
oh awesome! then I can change that! sorry thought the env.DOCKER_IMAGE set the runtime env